CONTRIBUTING.md

```diff
@@ -1,85 +1,85 @@
-# Contributing to the project
+# Contributing to the Project
 
-Hi 👋 Thanks for considering contributing! Aim of this doc it to guide you throught the process and help you better understand the project.
+Hi 👋 Thanks for considering contributing! The aim of this doc is to guide you throughout the process of contributing to and help you better understand the Style Guide project.
 
 ## Process
 
-1. Run this project locally (described below)
-1. Make your changes (including tests and documentation updates)
-1. Make sure all tests and linters are happy with your changes
-1. Open a PR
-1. Get your PR accepted by two of the [maintainers](https://github.com/brainly/style-guide/blob/master/MAINTAINERS)
-1. Bump project version (described below)
-1. Your chagne will be automatically deployed and available at [styleguide.brainly.com](https://styleguide.brainly.com) within couple of minutes
+1. Run this project locally (described below).
+2. Make your changes (including tests and documentation updates).
```

Comment:
`1.` was creating ordered list by default :)

Reply:
@aju Yes, I realize that, but is there a reason _not_ to use 1, 2, 3, ...? Sorry for the late response, by the way.